specs/conductor.md

```diff
@@ -1,59 +1,137 @@
+    - The `Message` value within the `GossipsubMessage` contains the Sequencer
+      block data from the Astria Sequencer. The `Message` is then parsed to a
+      `SequencerBlockData`
+      ([link](https://github.com/astriaorg/astria/blob/6e71a76fa52c522ffdcabcd9d659e4de765d9d61/crates/astria-sequencer-types/src/sequencer_block_data.rs#L39)).
+      Once transformed, the block data is validated to make sure that the
+      validator set correctly agreed on the block, the proposer for the Sequencer
+      and the block match, and that the previous commit hash in the block header
+      is correct.
+      The block is then passed to the Executor actor as a
+      `ExecutorCommand::BlockReceivedFromGossipNetwork` message which will
+      ultimately process and filter the block.
+- `DriverCommand::GetNewBlocks`([link](https://github.com/astriaorg/astria/blob/3c4e47dbe1818e4228691d6bfd2b2143a06f1a6e/crates/astria-conductor/src/driver.rs#L54))
+    - This message triggers the sending of a `ReaderCommand::GetNewBlocks` to the
+      Reader actor to initiate the pulling of data from the DA layer.
 
 ### Reader
- - Creates a `CelestiaClient` using the Celestia client implementation from `astria-sequencer-relayer` to communicating with the DA layer
- - Creates a `TendermintClient` which is used when validating blocks from the DA
-   layer against the sequencer data
- - Runs an event loop that handles receiving `ReaderCommand`s that drives data retrieval from the DA layer
- - Passes the blocks it receives to the `Executor`
+
+- Creates a `CelestiaClient` using the Celestia client implementation from `astria-sequencer-relayer` to communicate with the DA layer
+- Creates a `TendermintClient` which is used when validating blocks from the DA
+  layer against the Sequencer data
+- Runs an event loop that handles receiving `ReaderCommand`s that drive data retrieval from the DA layer
+- Passes the blocks it receives to the `Executor`
+
+The Reader receives a `ReaderCommand::GetNewBlocks`
+([link](https://github.com/astriaorg/astria/blob/3c4e47dbe1818e4228691d6bfd2b2143a06f1a6e/crates/astria-conductor/src/driver.rs#L54))
+message from the driver. The `CelestiaClient`
+([link](https://github.com/astriaorg/astria/blob/3c4e47dbe1818e4228691d6bfd2b2143a06f1a6e/crates/astria-sequencer-relayer/src/data_availability.rs#L244))
+is then called from the Reader to get data from the Celestia DA. This data is
+then parsed from a Celestia blob into individual blocks, each block is then validated to check that the validator set is correct and the
+proposer for the block matches the proposer of the Sequencer, and that the previous commit of the current block is correct.
+Each block is then transformed into a `SequencerBlockSubset`s and handed off to the Executor as a
+`ExecutorCommand::BlockReceivedFromDataAvailability` for transaction filtering, and passing off to the rollup for execution.
```

Comment:
the conductor has been updated to only pull relevant rollup txs, so there's no more transaction filtering